psycop/common/sequence_models/tests/test_embedding.py

```diff
@@ -76,10 +76,10 @@ def test_diagnosis_mapping(
             source_type="diagnosis",
             source_subtype="A",
         ),
-        # Check that diagnosis codes that are not in the mapping are excluded
```

Comment:
We should still test exclusion :-)

Reply:
This is already tested as the```Patient(
        patient_id=11,
        date_of_birth=dt.datetime(year=1990, month=1, day=1),
    ) ``` has temporal events with values "d1" or "d2" (which are not in the mapping). 
  
Adding it here would be redundant but maybe make it more clear that we are testing it?